Otec/NewspapersSubscriptionViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  NewspapersSubscriptionViewController.swift
+//  Otec
+//
+//  Created by El gera de la gente on 6/8/16.
+//  Copyright © 2016 Inaka. All rights reserved.
+//
+
+import UIKit
+
+class NewspapersSubscriptionViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+    @IBOutlet weak var tableView: UITableView!
+    
+    var newspaperSuscriptedIDs = UserNewspaperSession.newspapersSuscribed()
+    var allNewspapers = [Newspaper]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.newspaperSuscriptedIDs = UserNewspaperSession.newspapersSuscribed()
+        if self.checkIfComesFromFeed() {
+           self.navigationItem.rightBarButtonItem = nil
+        }
+
+        self.retrieveAndShowAllNewspapers()
+    }
+    
+    private func checkIfComesFromFeed() -> Bool {
```

Comment:
Naming it `comesFromFeed` should be enough. Given Swift conventions and standards, the word `check` in boolean functions is unnecessary.